extensions/amp-share-tracking/0.1/amp-share-tracking.js

```diff
@@ -43,6 +43,11 @@ export class AmpShareTracking extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
+    if (!this.isExperimentOn_()) {
+      getService(this.win, 'share-tracking',
+          () => Promise.reject(user().createError(TAG + ' disabled')));
```

Comment:
Resolve for the same reasoning as https://github.com/ampproject/amphtml/pull/4498#discussion_r74815193.